plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -15,20 +15,92 @@
 // limitations under the License.
 //
 
+global proc mayausdTranslatorExport_AnimationCB() {
```

Comment:
To be consistent with other places it would be better to use "mayaUsd"

Reply:
Such as the import where it has: mayaUsdTranslatorImport_XXX

Reply:
Sure.  I'm also planning on removing the "global" from most of these functions since it isn't needed.